### ISSUE TITLE
rclone selfupdate broken in v1.64.0 and v1.64.1

### ISSUE THREAD

**Issue by ncw** [rclone/rclone#7373]
```
$ sudo rclone selfupdate --stable -vv
2023/10/17 18:42:20 DEBUG : rclone: Version "v1.64.0" starting with parameters ["rclone" "selfupdate" "--stable" "-vv"]
2023/10/17 18:42:30 DEBUG : downloaded release archive with hashsum 28b8907df12cb866c627f7dd3a692326e073384ceb5e99328007941026bb73b8 from https://downloads.rclone.org/v1.64.1/rclone-v1.64.1-osx-amd64.zip
2023/10/17 18:42:30 DEBUG : downloaded hashsum list: https://downloads.rclone.org/v1.64.1/SHA256SUMS
2023/10/17 18:42:30 Error: failed to update rclone: invalid hashsum signature
```

This is caused by the upgrade of the "github.com/ProtonMail/go-crypto/openpgp" library in 014acc902df47e3576ca3ef4667ba935e6955840 by @henrybear327 . Which is in turn caused by this commit https://github.com/ProtonMail/go-crypto/commit/5503f24f010878669817f6dd4fc0d2bccd67bcd0

I have reported an upstream issue about this here https://github.com/ProtonMail/go-crypto/issues/187

Options to fix are
- upstream fix
- revert to a previous version of the go-crypto library. However this affects the protondrive backend causing it not to compile when I tried reverting it to the known good version.
- go back to using `x/crypto/openpgp`
- redo the signatures with the flag the openpgp library is now looking for

Note that you can test selfupdate without updating rclone like this `rclone selfupdate  --version v1.63.1 -vv --output /tmp/z`.

Note also that this wasn't picked up by the integration tests and it definitely should have been so that needs investigating.

**Comment by gh0sti** [comment]
I'm glad I did a Google search to find out why this broke and I was so confused as to why I was getting the hashsum error.

**Comment by ncw** [comment]
I'll release this fix in v1.64.2 tomorrow.

Unfortunately I can't fix v1.64.0 and v1.64.1 so if you've got one of those `selfupdate` won't work and you'll have to do a manual download.

**Comment by gh0sti** [comment]
> I'll release this fix in v1.64.2 tomorrow.
> 
> Unfortunately I can't fix v1.64.0 and v1.64.1 so if you've got one of those `selfupdate` won't work and you'll have to do a manual download.

Understandable. I'll just manually update.
